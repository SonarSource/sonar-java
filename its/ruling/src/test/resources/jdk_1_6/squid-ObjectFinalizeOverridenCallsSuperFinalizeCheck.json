{
'jdk_1_6:java/awt/image/IndexColorModel.java':[
1498,
],
'jdk_1_6:java/net/SocketInputStream.java':[
244,
],
'jdk_1_6:java/net/SocketOutputStream.java':[
159,
],
}
