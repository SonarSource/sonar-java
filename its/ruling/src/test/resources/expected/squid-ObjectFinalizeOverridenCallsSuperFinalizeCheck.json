{
'project:com/sun/imageio/plugins/common/SubImageInputStream.java':[
75,
],
'project:com/sun/imageio/plugins/png/PNGImageWriter.java':[
143,
260,
],
'project:java/awt/image/IndexColorModel.java':[
1515,
],
'project:java/net/SocketInputStream.java':[
265,
],
'project:java/net/SocketOutputStream.java':[
176,
],
'project:javax/imageio/stream/FileCacheImageInputStream.java':[
257,
],
'project:javax/imageio/stream/FileImageInputStream.java':[
159,
],
'project:javax/imageio/stream/FileImageOutputStream.java':[
167,
],
'project:javax/imageio/stream/MemoryCacheImageInputStream.java':[
182,
],
}
